src/pages/index.en.tsx

```diff
@@ -340,7 +334,12 @@ export const LandingPageFragment = graphql`
     contentfulHomePage(node_locale: { eq: $locale }) {
       landingLeadInText
       landingImage {
-        fluid {
+        fluid(quality: 100) {
```

Comment:
I didn't know about this quality option - is that something we should always use going forward or was it specific to this case?

Reply:
hmm i think if no one has mentioned the quality we can probably leave the other images as is for now? the default quality is 50 (https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-image/)